### ISSUE TITLE
Is it dead yet?

### ISSUE THREAD

**Issue by piotr-yuxuan** [keybase/client#24303]
https://github.com/keybase/client/graphs/commit-activity
![Screenshot_2020-08-28 keybase client commit activity](https://user-images.githubusercontent.com/3686502/91578657-63d15d00-e942-11ea-8f0e-27e0babf3b83.png)

**Comment by MarkWieczorek** [comment]
No. Your graph shows that there are commits being made, just not at the same pace as before.

**Comment by Ancillas** [comment]
I wonder where all of those other development cycles are going?  ;)

https://blog.zoom.us/zoom-acquires-keybase-and-announces-goal-of-developing-the-most-broadly-used-enterprise-end-to-end-encryption-offering/

**Comment by impredicative** [comment]
Is it smart for a new user to use Keybase? I'd like to hear from a developer.

**Comment by junderw** [comment]
They're working on Zoom in exchange for Zoom paying Keybase's hosting costs.

**Comment by boehs** [comment]
![](https://e.reeee.ee/qmF1pA.png)

Just wanted to put out my disappointment in @zoom for killing keybase, it was such a amazing project and so sad to see it die. If a zoom employee reads this, please take this as a vote for revival. Nothing is like it and every day I get sadder when I see nothing happening. Keep doing good :)

**Comment by CeriJohn57** [comment]
Put this on Reddit Sent from my Samsung A50 Galaxy smartphone.
-------- Original message --------From: Evan Boehs ***@***.***> Date: 08/06/2021  07:42  (GMT+07:00) To: keybase/client ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [keybase/client] This project is now dead (#24303) 

Just wanted to put out my disappointment in @zoom for killing keybase, it was such a amazing project and so sad to see it die. If a zoom employee reads this, please take this as a vote for revival. Nothing is like it and every day I get sadder when I see nothing happening. Keep doing good :)

**Comment by piotr-yuxuan** [comment]
With the benefit of hindsight, @MarkWieczorek would you say it is still alive?

**Comment by piotr-yuxuan** [comment]
<img width="1196" alt="Screenshot 2022-01-24 at 11 57 36" src="https://user-images.githubusercontent.com/3686502/150779014-53923903-7232-4cf4-bd47-06ecf1bb3427.png">

**Comment by iminto** [comment]
yes ,it was dead, there is no new feature many years
